Sources/StreamChatUI/ChatChannelList/ChatChannelListVC+SwiftUI.swift

```diff
@@ -0,0 +1,31 @@
+//
+// Copyright © 2021 Stream.io Inc. All rights reserved.
+//
+
+import StreamChat
+import SwiftUI
+
+@available(iOS 13.0, *)
+/// A `UIViewControllerRepresentable` subclass which wraps `ChatChannelListVC` and shows list of channels.
+public typealias ChatChannelListComponent = ChatChannelListVCComponent<NoExtraData>
+
+/// A `UIViewControllerRepresentable` subclass which wraps `ChatChannelListVC` and shows list of channels.
+public struct ChatChannelListVCComponent<ExtraData: ExtraDataTypes>: UIViewControllerRepresentable {
```

Comment:
Same question here - maybe it would make sense to namespace this? So we would have something like `ChatChannelListVC.View`? 

💡 Or maybe we could go all-in SwiftUI style where there are no views and view controller and use something like `ChatChannelList`? cc @GetStream/ios-developers

Reply:
I have name-spaced it as suggested, let's see how it feels.